modules/develop/pages/integrations.adoc

```diff
@@ -0,0 +1,343 @@
+
+
+| Couchbase Shell (cbsh)
+| Interactive shell, based upon nushell, which you can use to extract, transform, and load different data formats in Couchbase.
+a| 
+* https://couchbase.sh/docs/[cbsh docs]
+* https://github.com/couchbaselabs/couchbase-shell[GitHub repo]
+| ✔
+| ✔ 
+| Couchbase Officially Supported (currently in _Developer Preview_)
+
+
+| cbimport
+| Utilitly for importing data into Couchbase databases.
+Included in self-managed Couchbase Server, or downloadable as part of the xref:cloud:reference:command-line-tools.adoc#download-and-install-the-couchbase-command-line-tools[Capella Tools package].
+a| xref:server:tools:cbimport.adoc[cbimport reference]
+| ✔
+| ✔ 
+| Couchbase Officially Supported
+
+
+| cbexport
+| Utilitly for exporting data from Couchbase databases. 
+Included in self-managed Couchbase Server, or downloadable as part of the xref:cloud:reference:command-line-tools.adoc#download-and-install-the-couchbase-command-line-tools[Capella Tools package].
+a| xref:server:tools:cbexport.adoc[cbexport reference]
+| ✔
+| ✔ 
+| Couchbase Officially Supported
+
+
+| cbbackupmgr
+| A utility for backing up and restoring a Couchbase database.
+a| 
+* xref:server:current:backup-restore:cbbackupmgr.adoc[cbbackupmgr reference]
+* xref:server:backup-restore:cbbackupmgr-tutorial.adoc[Tutorial]
+* xref:cloud:clusters:cli-backup-restore.adoc[Capella examples]
+| ✔
+| ✔
+| Couchbase Officially Supported
+|===
+
+
+=== Other Shells
+
+Some languages, such as Ruby, come with REPLs (interactive shells) as standard -- as is normal with many programming languages.
+Others have them available as _Notebooks_, such as Python, with https://jupyter.org/[Jupyter Notebook] (which works with _several_ different languages), through which you can interactively develop functionality for a Couchbase app, or blend with analytic tools and charts.
+
+
+
+[big-data-connectors]
+== Big Data Connectors
+
+
+Couchbase develops and supports connectors for:
+
+* Elasticsearch and OpenSearch
+* The Kafka Connect Framework
+* Apache Spark Data Processing Platform
+* The Tableau interactive data visualization platform
+
+Couchbase ODBC and JDBC drivers enable any application based on the ODBC/JDBC standards, for example Microsoft Excel, QlikView, or SAP Lumira, as well as Tableau, to connect to a Couchbase Server or cluster.
+
+
+
+.Big Data Connectors
+[cols="26,45,30,30,30,28"]
+|===
+| Connector | Summary | Docs | Source | Other Integrations | <<#support-model,Support Model>>
+
+
+| Elasticsearch
+| Replicate your documents from Couchbase Server or Capella to Elasticsearch or OpenSearch in near real time. 
+| xref:elasticsearch-connector:getting-started.adoc[Getting Started docs]
+| https://docs.couchbase.com/elasticsearch-connector/current/release-notes.html[Release notes & download links]
+| https://docs.couchbase.com/elasticsearch-connector/current/autonomous-operations.html[AO Mode]
```

Comment:
Should these be xrefs or links? They're in the docs?

Reply:
:+1:

Reply:
:+1: